google/resource_endpoints_service.go

```diff
@@ -193,11 +220,11 @@ func resourceEndpointsServiceUpdate(d *schema.ResourceData, meta interface{}) er
 		source = getOpenAPIConfigSource(openapiConfig.(string))
 	} else {
 		grpcConfig, gok := d.GetOk("grpc_config")
-		protocOutput, pok := d.GetOk("protoc_output")
+		protocOutput, pok := d.GetOk("protoc_output_base64")
```

Comment:
this should probably still allow using `protoc_output` until the field is totally removed

Reply:
Done.